docs/overrides/main.html

```diff
@@ -1,15 +1,15 @@
 {% extends "base.html" %}
 
 {% block announce %}
-On March 25st, 2025 v2 of Powertools for AWS Lambda (Python) <a
-  href="https://github.com/aws-powertools/powertools-lambda-python/issues/5239" target="_blank">will reach
+On March 25th 2025, v2 reaches <a href="https://github.com/aws-powertools/powertools-lambda-python/issues/5239"
+  target="_blank">
   End-of-Life</a>. We recommend you to <a href="https://docs.powertools.aws.dev/lambda/python/latest/upgrade/"
   target="_blank">upgrade to v3</a>.
 {% endblock %}
 
 {% block outdated %}
-  You're not viewing the latest version.
-  <a href="{{ '../' ~ base_url }}">
-    <strong>Click here to go to latest.</strong>
-  </a>
+You're not viewing the latest version.
+<a href="{{ '../' ~ base_url }}">
+  <strong>Click here to go to latest.</strong>
+</a>
```

Comment:
np: accidental formatting on save that stripped whitespace; happy to revert if you need.

Reply:
Don't need!